flash/core/utilities/flash_cli.py

```diff
@@ -192,12 +191,9 @@ def parse_arguments(self) -> None:
     def add_arguments_to_parser(self, parser) -> None:
         subcommands = parser.add_subcommands()
 
-        inputs = self.local_datamodule_class.input_transform_cls().available_inputs()
-
-        for input in inputs:
-            if isinstance(input, InputFormat):
-                input = input.value
-            function = f"from_{input}"
+        for function in vars(self.local_datamodule_class).keys():
+            if not function.startswith("from"):
```

Comment:
DataModule also defines `from_argparse_args` (and maybe some others?), we could filter it out here

Reply:
* actually we are already checking with `is_overridden` so shouldn't be a problem